.github/workflows/recipe-catalog-change-template.yaml

```diff
@@ -0,0 +1,285 @@
+          -e TARGET_HOST_KEY_PATH=/data/id_rsa \
+          -e TARGET_FOLDER=pd-e2e \
+          -e TARGET_CLEANUP=false \
+          -e TARGET_RESULTS=results \
+          -e OUTPUT_FOLDER=/data \
+          -e DEBUG=true \
+          -v $PWD:/data:z \
+          quay.io/odockal/pde2e-podman:${{ env.PDE2E_IMAGE_VERSION }} \
+            pd-e2e/podman.ps1 \
+              -downloadUrl ${{ env.PODMAN_URL }} \
+              -targetFolder pd-e2e \
+              -resultsFolder results \
+              -initialize 0 \
+              -rootful 0 \
+              -start 0 \
+              -installWSL 0
+        # check logs
+        podman logs -f pde2e-podman-run
+
+    - name: Build Podman Desktop Electron Inspect Enabled binary
+      run: |
+        podman run --rm -d --name pde2e-builder-run \
+          -e TARGET_HOST=$(cat host) \
+          -e TARGET_HOST_USERNAME=$(cat username) \
+          -e TARGET_HOST_KEY_PATH=/data/id_rsa \
+          -e TARGET_FOLDER=pd-e2e \
+          -e TARGET_CLEANUP=false \
+          -e TARGET_RESULTS=results \
+          -e OUTPUT_FOLDER=/data \
+          -e DEBUG=true \
+          -v $PWD:/data:z \
+          quay.io/odockal/pde2e-builder:${{ env.PDE2E_IMAGE_VERSION }} \
+            pd-e2e/builder.ps1 \
+              -targetFolder pd-e2e \
+              -resultsFolder results \
+              -fork ${{ env.FORK }} \
+              -branch ${{ env.BRANCH }} \
+              -envVars ${{ env.ENV_VARS }}
+        # check logs
+        podman logs -f pde2e-builder-run
+
+    - name: Run Podman Desktop Playwright E2E tests
+      run: |
+        podman run -d --name pde2e-runner-run \
+          -e TARGET_HOST=$(cat host) \
+          -e TARGET_HOST_USERNAME=$(cat username) \
+          -e TARGET_HOST_KEY_PATH=/data/id_rsa \
+          -e TARGET_FOLDER=pd-e2e \
+          -e TARGET_RESULTS=results \
+          -e OUTPUT_FOLDER=/data \
+          -e DEBUG=true \
+          -v $PWD:/data:z \
+          quay.io/odockal/pde2e-runner:${{ env.PDE2E_IMAGE_VERSION }} \
+              pd-e2e/runner.ps1 \
+                -targetFolder pd-e2e \
+                -resultsFolder results \
+                -podmanPath $(cat results/podman-location.log) \
+                -pdPath "$(cat results/pde2e-binary-path.log | tr '\n' " ")" \
+                -fork ${{ env.FORK }} \
+                -branch ${{ env.BRANCH }} \
+                -extRepo ${{ env.EXT_REPO }} \
+                -extFork ${{ env.EXT_FORK }} \
+                -extBranch ${{ env.EXT_BRANCH }} \
+                -extTests ${{ env.EXT_TESTS }} \
+                -npmTarget ${{ env.NPM_TARGET }} \
+                -initialize ${{ env.PODMAN_INIT }} \
+                -rootful ${{ env.PODMAN_ROOTFUL }} \
+                -start ${{ env.PODMAN_START }} \
+                -userNetworking ${{ env.PODMAN_NETWORKING }} \
+                -envVars ${{ env.ENV_VARS }} \
+                -runAsAdmin ${{ env.PODMAN_ADMIN }} \
```

Comment:
env var is not set anywhere. I would not call it PODMAN, just RUN_AS_ADMIN. Remove the backslash.

Reply:
shouldn't this be parsed by this though?
```
        DEFAULT_PODMAN_OPTIONS: 'INIT=1,START=1,ROOTFUL=1,NETWORKING=0,ADMIN=1'
...
        echo "${{ env.DEFAULT_PODMAN_OPTIONS }}" | awk -F ',' \
         '{for (i=1; i<=NF; i++) {split($i, kv, "="); print "PODMAN_"kv[1]"="kv[2]}}' >> $GITHUB_ENV
```

Reply:
we should not hide it within podman setup parameters.

Reply:
ACK

Reply:
fixed